client/src/components/qa/AnswerList.jsx

```diff
@@ -53,8 +33,15 @@ function AnswerList({ answerList, getQuestions }) {
           helpfulness={answer.helpfulness}
```

Comment:
Why don't you pass the whole question object as a prop to your Answer component? Is it because the object is very big and bulky and it doesn't make sense to pass the whole object so you pass only relevant properties?